### PR TITLE
improves performance by eliminating feeder requests for ids

### PR DESCRIPTION
**Pull request by sandikbarr**
addresses #624 

I started to go the generic route with the guess id in the haldoc, but then I realized these actually weren't even hal links, they are url properties on a resource that we're treating like a link, and it was really looking like a static function in haldoc that didn't feel right, so I chose to go the specific route in the distribution models. Hopefully that's more of a guiding factor for future developers as well as guessing a link is not the recommended path (except in special cases I suppose.)

### CODE REVIEW

File: src/app/shared/model/distribution.model.ts [hunk 1]
```diff
@@ -86,6 +86,13 @@ export class DistributionModel extends BaseModel {
     return {podcast, versionTemplates};
   }
 
+  get feederPodcastId(): string {
+    if (this.kind === 'podcast' && this.url) {
+      const parts = this.url.match(/\/(.[^\/]*)$/);
```
Comment (cavis) [inline src/app/shared/model/distribution.model.ts]: This regex is a bit hard to follow.  Maybe more like a `this.url.split('/').pop()` or something?

File: src/app/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -33,18 +30,11 @@ export class DashboardComponent implements OnInit {
         this.defaultAccount = defaultAccount;
         return this.auth.followItems('prx:series', {filters: 'v4', zoom: 'prx:image'});
```
Comment (cavis) [inline src/app/dashboard/dashboard.component.ts]: I think if this zoom was `prx:image,prx:distributions` that would cut down on 10 more requests.

**Comment by sandikbarr** [comment]
Quality suggestions, thanks! Back to you @cavis
